src/views/HomeView.vue

```diff
@@ -104,13 +135,15 @@ export default {
   },
   data() {
     return {
+      tab: 'play',
```

Comment:
You should probably have consts in component for the valid tabs and use those instead.

Reply:
done